src/main/java/mate.academy/Main.java

```diff
@@ -10,7 +10,7 @@ public class Main {
     public static void main(String[] args) {
         // Please test your Injector here. Feel free to push this class as a part of your solution
```

Comment:
you need to remove this line description

Reply:
agree with @maksigors , let's remove redundant comments